wordpress-mobile/WordPress-iOS
# UL&S: Site Address flow: navigate to a blank SiteCredentialsViewController

Ref. #14009 
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/324

### To test:
1. Enable the .siteAddress feature flag
2. `rake dependencies`
3. Build and run
4. Log In > Or log in by entering your site address. > enter a valid URL, such as thuycopeland.com. Continue.
5. **Expected:** the continue button navigates you to the next screen. 

**iPhone SE screenshot**
<a href="https://user-images.githubusercontent.com/1062444/87069145-50283680-c1dc-11ea-8a9f-525e2237b6c3.png"><img src="https://user-images.githubusercontent.com/1062444/87069145-50283680-c1dc-11ea-8a9f-525e2237b6c3.png" width="350" /></a>


PR submission checklist:
